pkgs/applications/networking/irc/weechat/default.nix

```diff
@@ -1,8 +1,19 @@
-{ stdenv, fetchurl, ncurses, openssl, perl, python, aspell, gnutls
-, zlib, curl , pkgconfig, libgcrypt, ruby, lua5, tcl, guile
-, pythonPackages, cmake, makeWrapper, libobjc, libiconv
```

Comment:
while you're at it, would you mind making a second commit  where `pythonPackages` is removed, and we instead specify exactly which packages are needed.

In `all-packages.nix` the cleanest way to make sure you have the right packages is then to write `inherit (pythonPackages) python pycrypto;`